src/config/agoric/agoric.tsx

```diff
@@ -15,13 +15,15 @@ import {
 } from "../../lib/messageBuilder";
 import { isValidBundle } from "../../utils/validate";
 import { makeSignAndBroadcast } from "../../lib/signAndBroadcast";
+import { useWatchBundle } from "../../hooks/useWatchBundle";
 
 const Agoric = () => {
   const { netName } = useNetwork();
   const { walletAddress, stargateClient } = useWallet();
   const proposalFormRef = useRef<HTMLFormElement>(null);
   const corEvalFormRef = useRef<HTMLFormElement>(null);
   const bundleFormRef = useRef<HTMLFormElement>(null);
+  const watchBundle = useWatchBundle();
```

Comment:
`watchBundle` presumable access the net, yes? but no access to the net is passed around explicitly. What would it cost to remedy that lack of ocap discipline?

I suppose this goes for `useWallet` too.

Reply:
See [[c9c1537](https://github.com/DCFoundation/cosmos-proposal-builder/pull/39/commits/c9c1537aa6c26980325a6da0fd597b0ef00e4bad)]. `useWallet`, agree but deeming out of scope for this pr